app/src/main/res/font/metropolis.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+    <font-family xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto">
```

Comment:
Please add `tools:ignore="UnusedAttribute"` to the `font-family` element.  It'll suppress the lint errors.  The comment mentioned above can sit just below the suppression, explaining it.